docs/Cluster-VictoriaMetrics.md

```diff
@@ -1364,7 +1364,7 @@ Below is the output for `/path/to/vmstorage -help`:
      Supports an array of values separated by comma or specified via multiple flags.
   -retentionPeriod value
      Data with timestamps outside the retentionPeriod is automatically deleted. See also -retentionFilter
-     The following optional suffixes are supported: h (hour), d (day), w (week), y (year). If suffix isn't set, then the duration is counted in months (default 1)
+     The following optional suffixes are supported: h (hour), d (day), w (week), y (year). If suffix isn't set, then the duration is counted in months (default 1). The minimum retention period is 24h or 1d.
```

Comment:
I don't see the same change in flags description for README.md
also, you need to add to the flag description in the source code

Reply:
added clarifications to missed places